.github/workflows/ci.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         - { name: MacOS,                os: macos-11, flags: -GNinja }
         - { name: MacOS Xcode,          os: macos-11, flags: -GXcode }
         - { name: iOS,                  os: macos-11, flags: -DCMAKE_SYSTEM_NAME=iOS -DCMAKE_OSX_ARCHITECTURES=arm64 }
-        - { name: iOS Xcode,            os: macos-11, suffix: -- CODE_SIGNING_ALLOWED=NO, flags: -DCMAKE_SYSTEM_NAME=iOS -GXcode }
+        - { name: iOS Xcode,            os: macos-11, flags: -DCMAKE_SYSTEM_NAME=iOS -GXcode -DCMAKE_XCODE_ATTRIBUTE_CODE_SIGNING_ALLOWED=NO }
```

Comment:
Is there some official documentation for this variable? I tried to find the toolset, but couldn't directly.

Reply:
I couldn't find docs for `CMAKE_XCODE_ATTRIBUTE_CODE_SIGNING_ALLOWED` in particular. There appear to be many Xcode attributes so they all get lumped into this one page with the assumption that using them requires understanding the particular Xcode attribute you want.

https://cmake.org/cmake/help/latest/variable/CMAKE_XCODE_ATTRIBUTE_an-attribute.html